ocsp.go

```diff
@@ -517,7 +498,7 @@ func verifyPeerCertificate(verifiedChains [][]*x509.Certificate) (err error) {
 		}
 		results := getAllRevocationStatus(verifiedChains[i])
```

Comment:
Does this mean that the Root CA Certs will also be verified?

Reply:
No, it won't. The root CA is appended as the issuer cert to validate the subject.